.changeset/gorgeous-ads-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@team-plain/typescript-sdk': minor
```

Comment:
can these be one changeset?

Reply:
The could be but I made them as I went and this will generate a nicer changelog I think!